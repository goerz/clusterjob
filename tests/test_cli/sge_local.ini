[Attributes]
backend = sge
shell = /bin/bash
rootdir = /home/goerz
workdir = cjtest

[Resources]

time = 00:05:00
-j = y
-S = /bin/bash
