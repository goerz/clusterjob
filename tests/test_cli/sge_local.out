####################################################
Job id         : 2418
Job name       : test_clj
Workdir        : /home/goerz/cjtest
Submission Host: bigboy
Compute Node   : bigboy
Job started on bigboy Mon Feb 29 13:40:11 PST 2016
Current directory: /home/goerz/cjtest
####################################################
Job Finished:  Mon Feb 29 13:41:11 PST 2016
